src/customHooks/useLeaderBoard/useLeaderBoard.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@tanstack/react-query";
+import { getLeaderBoardFromLocalStorage } from "../../api/localStorageApi";
+import { errorMessage } from "./strings";
+
+
+export function useLeaderBoard(gameId: number) { 
+    const { data } = useQuery([gameId.toString], getLeaderBoardFromLocalStorage, {
+        onError: (err) => {
+            throw new Error(errorMessage + err)
+        },

```

Comment:
this will cause the entire page to crush. use the `isError` flag and if it's true, display an error message to the user :)